Episodes/24/description.md

```diff
@@ -0,0 +1,26 @@
+# Folge 24
+
+## Atlas UI
+
+E24 - 5Minutes - Atlas UI
+
+## Folgen Beschreibung
+
+In der 24. Folge unseres 5Minds Podcast geht es um unsere Atlas UI eine Komponente der Atlas Platform das neue Produkt der 5Minds.
+
+Die AtlasUI ist eine Rahmenanwendung benutzerorientierte Applikationen. Sie dient als Schnittstelle zwischen Benutzer und Prozess.
+
+![Screenshot AtlasUI](./atlasui-screen.png)
```

Comment:
Leider kann ich in der Folgen Beschreibung kein Bild einbinden.
Willst du es als zusätzlichen Satz unten drunter schreiben und auf GH verlinken?